filetablemodel.cpp

```diff
@@ -16,7 +17,7 @@ FileTableModel::FileTableModel(QObject *parent)
     : QAbstractTableModel(parent)
 {
     // We make it as object field because we could not make class field.
-    m_columns << tr("User") << tr("Password") << tr("Hash") << tr("GECOS");
+    m_columns << tr("User") << tr("Password") << tr("Hash") << tr("GECOS") << tr("Possible Hash Types");
```

Comment:
On Fri, May 22, 2015 at 06:49:40AM -0700, Shinnok wrote:
> > @@ -16,7 +17,7 @@ FileTableModel::FileTableModel(QObject *parent)
> >      : QAbstractTableModel(parent)
> >  {
> >      // We make it as object field because we could not make class field.
> > -    m_columns << tr("User") << tr("Password") << tr("Hash") << tr("GECOS");
> > +    m_columns << tr("User") << tr("Password") << tr("Hash") << tr("GECOS") << tr("Possible Hash Types");
> 
> Remove possible.

I am not sure... "Valid as Hash Types" does not seem good too.

Also "Types" maybe replaced by "Formats" because format is what used
at options tab. Also one "type" may have several "formats" (different
implementations).

-- 
Regards,
Aleksey Cherepanov